src/commands/build.rs

```diff
@@ -9,21 +9,21 @@ use super::up::UpCommand;
 
 /// Run the build command for each component.
 #[derive(Parser, Debug)]
-#[clap(about = "Build the Spin application", allow_hyphen_values = true)]
+#[command(about = "Build the Spin application")]
 pub struct BuildCommand {
     /// Path to spin.toml.
-    #[clap(
-            name = APP_CONFIG_FILE_OPT,
-            short = 'f',
-            long = "file",
-        )]
+    #[arg(
+        short = 'f',
+        long = "file",
+        id = APP_CONFIG_FILE_OPT,
+    )]
     pub app: Option<PathBuf>,
 
     /// Run the application after building.
-    #[clap(name = BUILD_UP_OPT, short = 'u', long = "up")]
+    #[arg(id = BUILD_UP_OPT, short, long)]
```

Comment:
It looks like clap has chosen `-U` and `--UP` here for some reason:

<img width="485" alt="Screenshot 2023-01-30 at 4 24 15 PM" src="https://user-images.githubusercontent.com/9322017/215619581-aca187be-3e4e-4a42-b3e7-1bc120ee598a.png">

Can we figure out why and fix?  (Or perhaps go back to setting it explicitly to `-u` and `--up`?)

Reply:
Oops. Forgot to reinclude when re-writing. Same as above. Clap keeps the uppercase from the const.